lib/fluent/plugin/out_http_shadow.rb

```diff
@@ -24,7 +24,7 @@ def initialize
     config_param :username, :string, :default => nil
     config_param :password, :string, :default => nil
     config_param :rate, :integer, :default => 100
-    config_param :rate_per_method_hash, :hash, :default => nil
+    config_param :rate_per_host_hash, :hash, :default => nil
```

Comment:
I love YAGNI, too. :smile: 
However one downside I could imagine is, 

1. There might be a bug or improvement in upstream and it's done in `toyama0919/fluent-plugin-http_shadow`.
1. Then if `rate_per_method_hash` is also changed, it's always hard to rebase.